README.md

```diff
@@ -1347,6 +1347,12 @@ as a configuration object.
 `cwd` can optionally be provided, the package.json will be read
 from this location.
 
+.recommendCommands(boolean)
```

Comment:
We should probably note that the boolean arg is optional, and calling this without it enables the functionality.